rawkv/rawkv.go

```diff
@@ -249,13 +249,16 @@ func (c *Client) BatchPut(ctx context.Context, keys, values [][]byte) error {
 	if len(keys) != len(values) {
 		return errors.New("the len of keys is not equal to the len of values")
 	}
+	if nil != ttls && len(keys) != len(ttls) {
```

Comment:
```suggestion
	if len(ttls) > 0 && len(keys) != len(ttls) {
```
`nil != ttls` is not very easy to read.